controllers/utils/zones.go

```diff
@@ -75,6 +75,17 @@ func (r *ReconciliationRunner) GetZones(zones *infrav1.CloudStackZoneList) Cloud
 			client.MatchingLabels(capiClusterLabel),
 		); err != nil {
 			return ctrl.Result{}, errors.Wrap(err, "failed to list zones")
+		}
+		return ctrl.Result{}, nil
+	}
+}
+
+// GetZonesAndRequeueIfMissing gets CloudStackZones owned by a CloudStackCluster via an ownership label, and requeues
```

Comment:
nit: i'd avoid the implementation detail in the comment about _how_ CloudStackZones are retrieved

```suggestion
// GetZonesAndRequeueIfMissing gets CloudStackZones owned by a CloudStackCluster, and requeues
```